internal/provider/gql.go

```diff
@@ -598,6 +627,18 @@ type Integration struct {
 	ConfigurationOptions ClientIntegrationConfigurationOptions `graphql:"configurationOptions"`
 }
 
+// SpaceID returns the space where the integration is configured (using the integration MRN).
+func (i Integration) SpaceID() string {
```

Comment:
nit: i believe cnquery has code to manage mrns and to be able to obtain parts (resources) from it. might remove the need for this logic

Reply:
Would love to know where that lives and use it. We can do the swap later.